gen/definitions/high_availability.yaml

```diff
@@ -0,0 +1,145 @@
+---
+name: Device HA Pairs
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/devicehapairs/ftddevicehapairs
+data_source_name_query: true
+doc_category: Device
+attributes:
+  - model_name: name
+    type: String
+    mandatory: true
+    description: The name of the access control policy.
+    example: FTD_HA
+  - model_name: type
+    type: String
+    value: DeviceHAPair
```

Comment:
can you please make this field `computed` instead of `value`?
Also, this field is missing the description (which can also contain type)

Reply:
Done

Reply:
Breaking the code with just computed:true
Other features have also used the value field for type. Thus keeping this

Reply:
`computed` is relatively new feature in the template, so most of the objects are not yet migrated to it. Though if you are saying that making this compute breaks the resource, then it should stay this way.